test/translate_ligo.coffee

```diff
@@ -72,10 +72,10 @@ describe "translate ligo section unsorted", ()->
     const enumType_TWO : nat = 2n;
     (* enum EnumType converted into list of nats *)
 
-    function ternary (const self : state) : (list(operation) * state) is
+    function ternary (const #{config.reserved}__unit : unit) : (unit) is
```

Comment:
I propose to call this argument just `dummy` or `dummy_arg` everywhere

Reply:
This translation was before I introduced my changes